Explorer/Assets/DCL/Roads/Settings/RoadSettingsAsset.cs

```diff
@@ -14,5 +17,100 @@ public class RoadSettingsAsset : ScriptableObject, IRoadSettingsAsset
 
         IReadOnlyList<RoadDescription> IRoadSettingsAsset.RoadDescriptions => RoadDescriptions;
         IReadOnlyList<AssetReferenceGameObject> IRoadSettingsAsset.RoadAssetsReference => RoadAssetsReference;
+
+#if UNITY_EDITOR
+
+        /// <summary>
+        /// A custom inspector that just shows a search box to make finding road descriptions easier.
+        /// </summary>
+        [CustomEditor(typeof(RoadSettingsAsset))]
+        private class RoadSettingsAssetEditor : Editor
+        {
+            private Vector2Int coordinatesToSearch;
+            private int foundElementIndex = -1;
+            private Vector2Int foundElementCoordinates;
+            private Vector3 foundElementRotation;
+            private string foundElementModel;
+            private bool showNotFoundMessage;
+            private bool showUpdatedMessage;
+
+            public override void OnInspectorGUI()
+            {
+                Color originalColor = GUI.contentColor;
+                GUI.contentColor = Color.cyan;
+
+                EditorGUILayout.BeginVertical();
+                {
+                    EditorGUILayout.LabelField("Search for a Road description by its coordinates:");
+
+                    EditorGUI.BeginChangeCheck();
+                    {
+                        coordinatesToSearch = EditorGUILayout.Vector2IntField(GUIContent.none, coordinatesToSearch);
+                    }
+                    if (EditorGUI.EndChangeCheck())
+                    {
+                        foundElementIndex = -1;
+                        showUpdatedMessage = false;
+                        showNotFoundMessage = false;
+                    }
+
+                    if (GUILayout.Button("Search"))
+                    {
+                        List<RoadDescription> descriptions = (target as RoadSettingsAsset).RoadDescriptions;
```

Comment:
Consider casting target to the correct type once at the start of the functions and using that throughout.